TypeCobol/Compiler/Diagnostics/CrossChecker.cs

```diff
@@ -12,6 +12,7 @@
 using TypeCobol.Compiler.Directives;
 using TypeCobol.Compiler.Scanner;
 using TypeCobol.Compiler.Symbols;
+using String = System.String;
```

Comment:
Other using directives have not changed so this is certainly not needed. In any case if `String` is ambiguous, you can (and should 😄) use `string` keyword.

Reply:
Done